models/keyword.go

```diff
@@ -18,9 +20,21 @@ const (
 	Processed  KeywordStatus = "processed"
 	Failed     KeywordStatus = "failed"
 
-	InvalidKeywordStatusErr = "invalid keyword status"
+	InvalidKeywordStatusErr    = "invalid keyword status"
+	couldNotJoinConditionError = "could not join conditions"
 )
 
+type Condition struct {
```

Comment:
As an improvement, since this struct could be used in other models, this struct and the method `getJoinedConditions` could be moved to another file in `models` package

Reply:
Good catch. Fixed in a1d1a1d1ffd6fb5cb7ad53c944104ca1c82e97d9